vic5_manuscript.tex

```diff
@@ -252,7 +256,9 @@ \section{VIC Infrastructure}
       \item \textit{Performance}: With the addition of parallel processing to the \textit{Image} and \textit{CESM drivers}, a set of tools for assessing the parallel performance were needed. The performance tests assess timing and memory statistics of the VIC model. Results from these tests are highlighted in Sect. \ref{sec:results}.
     \end{enumerate}
 
-    The \textit{Build}, \textit{Unit}, \textit{System}, and \textit{Examples} tests are routinely run using the Travis CI continuous integration system (\url{https://travis-ci.org/UW-Hydro/VIC}) where the VIC test suite is executed using a variety of compilers and computational architectures. The VIC's Travis workflow includes the following steps: 1) selection of virtual machine with preinstalled environments, 2) download of VIC source code and test datasets from GitHub, 3) compile specific driver along with Python unit tests, and 4) run series of tests for specific driver implementation (e.g. image). If any of the tests in these steps fail for any part of the test matrix, Travis returns a failed test status to the VIC GitHub page. This functionality is routinely used in evaluating VIC modifications that are suggested on GitHub via pull requests.
+    The \textit{Build}, \textit{Unit}, \textit{System}, and \textit{Examples} tests are routinely run using the Travis CI continuous integration system (\url{https://about.travis-ci.com/}, \url{https://travis-ci.org/UW-Hydro/VIC}) where the VIC test suite is executed using a multiple of compilers (GNU and Clang), MPI libraries (MPICH and Open-MPI) and computational architectures (Linux vs OS X). The VIC's Travis workflow includes the following steps: 1) selection of virtual machine with preinstalled environments, 2) download of VIC source code and test datasets from GitHub, 3) compile specific driver along with Python unit tests, and 4) run series of tests for specific driver implementation (e.g. image). If any of the tests in these steps fail for any part of the test matrix, Travis returns a failed test status to the VIC GitHub page. This functionality is routinely used in evaluating VIC modifications that are suggested on GitHub via pull requests.
+
+    % TODO: mention that vic has been tested with PGI, Intel, Cray compilers but they can't be used on travis because of license issues.

```

Comment:
Go ahead and complete all the TODOs - I can then check out this PR locally and make one more round of edits (and issue a PR against this PR)